README.md

```diff
@@ -334,7 +336,7 @@ Usually when you reference `process.env.FOO` in your code, webpack will replace
 
 **But:** Since Hops renders applications on the server- and client-side, we can reference environment variables in the settings and pass them on to the client and therefore allow users to have environment variables that are evaluated at runtime.
```

Comment:
This needs a hint, that exposing those values to the client-side needs an additional effort.